library/src/main/java/com/digi/xbee/api/CellularDevice.java

```diff
@@ -168,6 +168,25 @@ public void readDeviceInfo() throws TimeoutException, XBeeException {
 	}
 	
 	/**
+	 * Returns whether the device is connected to the Internet or not.
+	 * 
+	 * @return {@code true} if the device is connected to the Internet, 
+	 *         {@code false} otherwise.
+	 * 
+	 * @throws InterfaceNotOpenException if this device connection is not open.
+	 * @throws TimeoutException if there is a timeout getting the association 
+	 *                          indication status.
+	 * @throws XBeeException if there is any other XBee related exception.
+	 * 
+	 * @see #getCellularAssociationIndicationStatus()
+	 * @see com.digi.xbee.api.models.CellularAssociationIndicationStatus
+	 */
+	public boolean isConnected() throws TimeoutException, XBeeException {
```

Comment:
This method is also available in the ```WiFiDevice``` class and both classes (```WiFiDevice``` and ```CellularDevice```) extend the ```IPDevice``` one. Shouldn't the ```isConnected``` method be abastract?